src/modules/learning-object-service/objects/ObjectsController.ts

```diff
@@ -323,7 +323,7 @@ export class ObjectsController implements Controller {
 
     /**
      * @swagger
-     * /learning-objects:
+     * /learning-objects?assigned:
```

Comment:
I don't think we need to add the query parameter here. The comments below for the docs are enough.

Reply:
^